src/operand/__test__/unit/reference.test.ts

```diff
@@ -30,7 +38,24 @@ describe('Operand - Value', () => {
           subB: {
             subSubA: 3
           }
-        }
+        },
+        RefD: 'A',
+        RefE: 'D',
+        RefF: 'subA',
+        RefG: [
+          'Apples',
+          'Oranges',
+          'Fish'
+        ],
+        RefH: [
+          {
+            subA: 1
+          },
+          {
+            subA: 2
+          }
+        ],
+        Refundefined: 'A'
```

Comment:
Let's remove this alien :)

Reply:
This was intentional though. My code was resolving the reference to `Refundefined`. It was returning undefined because `Refundefined` wasn't part of the context. So I added it, to make the test fail. Then I fixed the code to make the test pass again.

Reply:
03f6ece2d37ace1a91007c78367f9d73ae381548